docs/configuration/v2.x/metrics/express-route-circuits.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Azure Express Route Circuits Declaration
+---
+
+## Azure Express Route Circuits
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.0-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
+
+You can declare to scrape an Azure Express Route Circuits (without Peerings) via the `ExpressRouteCircuits` resource
+type.
+
+The following fields need to be provided:
+
+- `expressRouteCircuitsName` - The name of the express route circuits
```

Comment:
I'm new to ExpressRoute but is this the name of the express route or expressroute circuit?

Reply:
```suggestion
- `expressRouteCircuitName` - The name of the Azure Express Route circuit
```

Reply:
Actually since it's the `ExpressRouteCircuit` scraper I'm even wondering if we can't call it `CircuitName`?

Reply:
I missed that. It's the name of the ExpressRoute Circuit

Reply:
Regarding the name change I'm pretty unemotional. The full name might be a bit clearer